client/components/post-schedule/utils.js

```diff
@@ -51,24 +51,24 @@ const getLocalizedDate = ( date, tz, gmt ) => {
 	return date;
 };
 
-const getDateInLocalUTC = date => moment( date.format ? date.format() : date );
+export const getDateInLocalUTC = date => moment( date.format ? date.format() : date );
```

Comment:
Doesn't seem to be used outside this file? Not sure if we want to `export` it in that case or treat it as a "private" method-type of thing?
Same for `getTimeOffset`.

Reply:
it won't hurt as an `export` and it was already being exported anyway 🤷‍♂️